Microsoft.Toolkit.Services/Services/MicrosoftGraph/MicrosoftGraphAuthenticationHelper.cs

```diff
@@ -180,20 +181,21 @@ public async Task<string> GetUserTokenV2Async(string appClientId, UIParent uiPar
         /// Logout the user
         /// </summary>
         /// <returns>Success or failure</returns>
-        public bool Logout()
+        public async Task<bool> Logout()
```

Comment:
This is a breaking change, we should create a new method called LogoutAsync instead